code/modules/mob/living/carbon/human/life.dm

```diff
@@ -78,6 +78,8 @@
 		handle_stamina()
 
 		if (is_diona())
+			if(health < -40) //240+ damage
+				diona_split_into_nymphs()
```

Comment:
That wont work. As the diona already splits at 0 health due to the death -> It will never be able to reach -40

Reply:
Not 100% what I was doing. Dionae health is back to 300, and they split at 60 health.